app/views/layouts/application.html.erb

```diff
@@ -4,10 +4,6 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
 
-    <% flash.each do |name, msg| %>
-      <%= content_tag(:div, msg, class: "alert alert-#{name}") %>
-    <% end %>
-
     <div class=flash_messages>
       <%= display_flash_messages %>
```

Comment:
I'm not familiar with this method! At the risk of sounding obtuse, what does it do?